### ISSUE TITLE
Add Images to the First Contribution tutorial

### ISSUE THREAD

**Issue by rcallaby** [rcallaby/Learn-Git#125]
**Is your feature request related to a problem? Please describe.**
Yes, in order to clarify the tutorial images of the process would greatly help in understanding of the problem

**Describe the solution you'd like**
Add screenshots of the process explained in the tutorial to help illustrate the solution

**Comment by RileeyL** [comment]
Hi @rcallaby , This will be one of my first ever contributions and I was hoping to be assigned to this task. I may however have a couple questions along the way as I am still learning, however this can help with the task you given above as I have discovered where adding images to the tutorial was extremely helpful. Please let me know if I may be assigned to this task, thank you so much!

**Comment by rcallaby** [comment]
@RileeyL It is all yours now

**Comment by rcallaby** [comment]
This is now open again.

**Comment by RileeyL** [comment]
Hi @rcallaby I think I would like to try again on the project, I now have a better understanding of what screenshots I can and should include and I think I have figured out the command prompt section. Would you let me try again? Thanks

**Comment by rcallaby** [comment]
@RileeyL Could you please select another open issue? Like Issue #133?
